pandas/tests/indexing/test_partial.py

```diff
@@ -84,37 +81,6 @@ def test_partial_setting(self):
         df.loc[3] = df.loc[2]
         tm.assert_frame_equal(df, expected)
 
-        # single dtype frame, overwrite
-        expected = DataFrame(dict({"A": [0, 2, 4], "B": [0, 2, 4]}))
-        df = df_orig.copy()
-        with catch_warnings(record=True):
-            df.ix[:, "B"] = df.ix[:, "A"]
```

Comment:
is this tested for `loc` ? (otherwise keep test but with loc? same for the ones below)